src/Ops.Web/Areas/SiteManagement/Views/Podcasts/EpisodeDetails.cshtml

```diff
@@ -196,4 +251,59 @@
             @Model.MaximumFileSizeMB MB
         </div>
     </div>
+    <div class="row form-group mt-4">
+        <div class="col-form-label text-md-right col-md-3">
+            <label asp-for="Episode.ShowNotesSegmentId">
+            </label>
+            <span class="fas fa-asterisk fa-xs d-inline-block ml-2 text-danger oc-required-field-marker">
```

Comment:
It seems like Show Notes aren't required so we might be able to remove this required asterisk, I might be wrong though.